Plugins/Reqnroll.CustomPlugin/README.md

```diff
@@ -0,0 +1,9 @@
+﻿Reqnroll.CustomPlugin provides a sample custom plugin for Reqnroll.
```

Comment:
It should be "Reqnroll.CustomPlugin provides the required dependencied for writing custom Reqnroll generator extensions."